printer_mods/KevinAkaSam/VEFACH/STL+CAD/stlcad.md

```diff
@@ -0,0 +1,3 @@
+All STL + CAD files.
+Please take a look at the instructions listed in the main README before priting.
```

Comment:
Replace `priting` with `printing`.

Reply:
Hey, thanks for your detailed feedback. I'm pretty new to GitHub and English is not my first language, so please forgive the mistakes I made. I have changed all the things you mentioned. I also answered your question because I thought it would be helpful for others. I also added a second solution without modifiers, I hope this is an acceptable solution :)